src/tools/clippy/tests/ui/async_yields_async.stderr

```diff
@@ -83,12 +91,14 @@ LL | |         println!("I'm bored");
 LL | |         // Some more stuff
 ...  |
 LL | |         CustomFutureType
-   | |         ^^^^^^^^^^^^^^^^
-   | |         |
-   | |         awaitable value not awaited
-   | |         help: consider awaiting this value: `CustomFutureType.await`
+   | |         ^^^^^^^^^^^^^^^^ awaitable value not awaited
 LL | |     };
    | |_____- outer async construct
+   |
+help: consider awaiting this value
+   |
+LL |         CustomFutureType.await
+   |
```

Comment:
Interesting, we're missing an underline here.

Reply:
Does this happen, because we're replacing the entire line? The clippy suggestion probably doesn't add the `.await`, but suggests to remove the `CustomFutureType` and add `CustomFutureType.await`.